bindings/rust/bench/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.svg
+.cargo
```

Comment:
Because for memory benching `use-awslc-rustls.sh` and `use-awslc-s2n.sh` change `.cargo/config.toml`, I don't want to check it in. If it's checked in an included on a fresh clone of the repo, the things in the `.cargo/config.toml` will point to invalid paths and will break the bench.